test/utils.js

```diff
@@ -3,7 +3,14 @@
 const async = require('async')
 const _ = require('lodash')
 const PeerId = require('peer-id')
+const PeerInfo = require('peer-info')
+const PeerBook = require('peer-book')
+const multiaddr = require('multiaddr')
 const Bitswap = require('../src')
+const libp2p = require('libp2p-ipfs')
+const os = require('os')
+const Repo = require('ipfs-repo')
+const fsBS = require('fs-blob-store')
```

Comment:
Maybe it's more efficient if we use the memory blob store for these tests, what do you think?